mnefun/_scoring.py

```diff
@@ -109,6 +109,7 @@ def _pick_events(events, picker):
     events = picker(events)
     new_cnt = len(events)
     print(f'  Using {new_cnt}/{old_cnt} events.')
+    return events
```

Comment:
... and fixed this regression from #323 :)

Reply:
Oh, good catch.  I apparently didn't thoroughly test having a proper 'picker' function.